lib/mlibrary_search_parser/transform/solr/local_params.rb

```diff
@@ -13,6 +13,7 @@ def transform!
           else
             super
             set_param("q", "_query_:#{query}")
+            set_param("clean_string", original_search_tree.clean_string)
```

Comment:
I think we don't need this any more

Reply:
Changing it to a method on solr transforms so you can do `localparams_obj.clean_string`, parallel with what we do with everything else.